src/patient-registration/patient-registration.component.tsx

```diff
@@ -117,18 +153,22 @@ export const PatientRegistration: React.FC = () => {
         ],
       },
     };
-
-    savePatient(abortController, patient).then(
-      response => response.status == 201 && history.push(`/patient/${response.data.uuid}/chart`),
-      createErrorHandler(),
-    );
+    savePatient(abortController, patient)
+      .then(response => response.status == 201 && history.push(`/patient/${response.data.uuid}/chart`))
+      .catch(response => {
+        if (response.responseBody.error.globalErrors) {
+          response.responseBody.error.globalErrors.forEach(error => {
+            showToast({ description: error.message });
+          });
+        }
```

Comment:
As stated below, this is the only blocking issue. Let's try

```suggestion
          });
        } else {
          createErrorHandler()(response);
        }
```

Please verify that this works, at least by checking that the error is handled correctly when you block that URL:

![Screenshot from 2020-08-25 14-16-51](https://user-images.githubusercontent.com/1031876/91228935-ec02f880-e6dd-11ea-865e-dca2cdfc5ee4.png)

Reply:
It works fine 👍